draft-detecting-unwanted-location-trackers.md

```diff
@@ -308,45 +308,47 @@ The Product Data operand represents an 8-byte value that is intended to serve as
 This value SHALL be determined during the [onboarding process](#onboarding).
 
 
-| Operand name         | Data type | Size (octets) | Description                         |
-|:--------------------:|:---------:|:-------------:|:-----------------------------------:|
-| Product Data         | Uint8     | 8             | See [Product data](#product-data)   |
+| Operand name         | Data type | Count | Total Size (Bytes) | Description                      |
+|:--------------------:|:---------:|:-----:|:------------------:|:---------------------------------:|
+| Product Data         | Uint8     | 8     |      64            | See [Product data](#product-data) |
 {: title="Product Data Operand" }
 
+
 #### Manufacturer name
 The Manufacturer Name operand contains the name of the company whose brand will appear on the accessory, e.g., ”Acme”.
 
 
-| Operand name  | Data type | Size (octets) |    Description    |
-|:--------------------:|:---------:|:-------------:|:-----------------:|
-| Manufacturer Name    | UTF-8     | 64            | Manufacturer name |
+| Operand name      | Data type | Count | Total Size (Bytes) | Description                      |
+|:-----------------:|:---------:|:-----:|:------------------:|:--------------------------------:|
+| Manufacturer Name | UTF-8     | 64    |        64          | Manufacturer name                |
```

Comment:
Should we somehow indicate that this is "at most 64"? Same for Model Name below.